packages/pluggableWidgets/toggle-buttons-native/src/ToggleButtons.editorConfig.ts

```diff
@@ -0,0 +1,48 @@
+import { RowLayoutProps, StructurePreviewProps } from "@mendix/piw-utils-internal";
+
+import { ToggleButtonsPreviewProps } from "../typings/ToggleButtonsProps";
+
+// import toggleButtonSvgDark from "./assets/ToggleButtonDefault.dark.svg";
```

Comment:
These comments can be removed

Reply:
I forgot to delete these lines, and also I deleted those SVG files as well. Thanks and done 👍